css/mixins.css

```diff
@@ -9,3 +9,7 @@
   width: 170px;
   height: 40px;
```

Comment:
These aren't mixins, the first one is a class selector, and the second one is a hover on a button